src/wallet/wallet2.cpp

```diff
@@ -4955,20 +4955,23 @@ bool wallet2::save_multisig_tx(const multisig_tx_set &txs, const std::string &fi
   return epee::file_io_utils::save_string_to_file(filename, ciphertext);
 }
 //----------------------------------------------------------------------------------------------------
-std::string wallet2::save_multisig_tx(const std::vector<pending_tx>& ptx_vector)
-{
-  multisig_tx_set txs;
-  txs.m_ptx = ptx_vector;
+wallet2::multisig_tx_set wallet2::make_multisig_tx_set(const std::vector<pending_tx>& ptx_vector) const {
+    multisig_tx_set txs;
```

Comment:
Please conform to the same indentation convention already used in each file, which in this case is two whitespaces.